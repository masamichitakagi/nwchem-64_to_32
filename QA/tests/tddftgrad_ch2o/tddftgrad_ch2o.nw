echo

start tddftgrad_ch2o

geometry nocenter
O    -0.00872487   -0.68498262    0.00000000
C     0.00401562    0.54500235    0.00000000
H    -0.93426300    1.15321853    0.00000000
H     0.95433234    1.13412348    0.00000000
symmetry c1
end

basis spherical
 * library "6-31G"
end

driver
 maxiter 100
end

dft
 iterations 500
 xc slater 1.0 vwn_5 1.0
 grid xfine
 grid euler
 tolerances tol_rho 1.0e-9
 direct
end

tddft
 nroots 2
 algorithm 1
 nosinglet
 target 1
 civecs
 grad
  root 1
 end
end

#print "tddft grad"
task tddft gradient
