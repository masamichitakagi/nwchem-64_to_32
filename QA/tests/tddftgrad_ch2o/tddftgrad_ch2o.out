 argument  1 = tddftgrad_ch2o.nw



============================== echo of input deck ==============================
echo

start tddftgrad_ch2o

geometry nocenter
O    -0.00872487   -0.68498262    0.00000000
C     0.00401562    0.54500235    0.00000000
H    -0.93426300    1.15321853    0.00000000
H     0.95433234    1.13412348    0.00000000
symmetry c1
end

basis spherical
 * library "6-31G"
end

driver
 maxiter 100
end

dft
 iterations 500
 xc slater 1.0 vwn_5 1.0
 grid xfine
 grid euler
 tolerances tol_rho 1.0e-9
 direct
end

tddft
 nroots 2
 algorithm 1
 nosinglet
 target 1
 civecs
 grad
  root 1
 end
end

#print "tddft grad"
task tddft gradient
================================================================================


                                         
                                         


              Northwest Computational Chemistry Package (NWChem) 6.5
              ------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2014
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname        = orion
    program         = ../../../bin/LINUX64/nwchem
    date            = Tue Sep  2 17:16:01 2014

    compiled        = Sat_Aug_30_06:49:11_2014
    source          = /home/niri/nwchem/nwchem-6.5
    nwchem branch   = 6.5
    nwchem revision = 26025
    ga revision     = 10501
    input           = tddftgrad_ch2o.nw
    prefix          = tddftgrad_ch2o.
    data base       = ./tddftgrad_ch2o.db
    status          = startup
    nproc           =        4
    time left       =     -1s



           Memory information
           ------------------

    heap     =   13107196 doubles =    100.0 Mbytes
    stack    =   13107201 doubles =    100.0 Mbytes
    global   =   26214400 doubles =    200.0 Mbytes (distinct from heap & stack)
    total    =   52428797 doubles =    400.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = .
  0 scratch   = .




                                NWChem Input Module
                                -------------------



 Scaling coordinates for geometry "geometry" by  1.889725989
 (inverse scale =  0.529177249)

 Turning off AUTOSYM since
 SYMMETRY directive was detected!


          ------
          auto-z
          ------
  Looking for out-of-plane bends


                             Geometry "geometry" -> ""
                             -------------------------

 Output coordinates in angstroms (scale by  1.889725989 to convert to a.u.)

  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 O                    8.0000    -0.00872487    -0.68498262     0.00000000
    2 C                    6.0000     0.00401562     0.54500235     0.00000000
    3 H                    1.0000    -0.93426300     1.15321853     0.00000000
    4 H                    1.0000     0.95433234     1.13412348     0.00000000

      Atomic Mass 
      ----------- 

      O                 15.994910
      C                 12.000000
      H                  1.007825


 Effective nuclear repulsion energy (a.u.)      30.7230945931

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
    -0.0484448265     0.1464445989     0.0000000000



                                Z-matrix (autoz)
                                -------- 

 Units are Angstrom for bonds and degrees for angles

      Type          Name      I     J     K     L     M      Value
      ----------- --------  ----- ----- ----- ----- ----- ----------
    1 Stretch                  1     2                       1.23005
    2 Stretch                  2     3                       1.11817
    3 Stretch                  2     4                       1.11811
    4 Bend                     1     2     3               122.35888
    5 Bend                     1     2     4               122.38910
    6 Bend                     3     2     4               115.25203
    7 Torsion                  1     2     3     4         180.00000
    8 Torsion                  1     2     4     3         180.00000
    9 Torsion                  3     2     1     4         180.00000
   10 Torsion                  1     4     2     3         180.00000
   11 Torsion                  3     1     2     4         180.00000
   12 Torsion                  1     3     2     4         180.00000


            XYZ format geometry
            -------------------
     4
 geometry
 O                    -0.00872487    -0.68498262     0.00000000
 C                     0.00401562     0.54500235     0.00000000
 H                    -0.93426300     1.15321853     0.00000000
 H                     0.95433234     1.13412348     0.00000000

 ==============================================================================
                                internuclear distances
 ------------------------------------------------------------------------------
       center one      |      center two      | atomic units |  angstroms
 ------------------------------------------------------------------------------
    2 C                |   1 O                |     2.32446  |     1.23005
    3 H                |   2 C                |     2.11303  |     1.11817
    4 H                |   2 C                |     2.11292  |     1.11811
 ------------------------------------------------------------------------------
                         number of included internuclear distances:          3
 ==============================================================================



 ==============================================================================
                                 internuclear angles
 ------------------------------------------------------------------------------
        center 1       |       center 2       |       center 3       |  degrees
 ------------------------------------------------------------------------------
    1 O                |   2 C                |   3 H                |   122.36
    1 O                |   2 C                |   4 H                |   122.39
    3 H                |   2 C                |   4 H                |   115.25
 ------------------------------------------------------------------------------
                            number of included internuclear angles:          3
 ==============================================================================



  library name resolved from: environment
  library file name is: </home/niri/nwchem/nwchem-6.5/src/basis/libraries/>
  


 Summary of "ao basis" -> "" (spherical)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 *                           6-31G                    on all atoms 


 tddft_input: filename not found; default name will be used

                                 NWChem DFT Module
                                 -----------------


                      Basis "ao basis" -> "ao basis" (spherical)
                      -----
  O (Oxygen)
  ----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  5.48467170E+03  0.001831
  1 S  8.25234950E+02  0.013950
  1 S  1.88046960E+02  0.068445
  1 S  5.29645000E+01  0.232714
  1 S  1.68975700E+01  0.470193
  1 S  5.79963530E+00  0.358521

  2 S  1.55396160E+01 -0.110778
  2 S  3.59993360E+00 -0.148026
  2 S  1.01376180E+00  1.130767

  3 P  1.55396160E+01  0.070874
  3 P  3.59993360E+00  0.339753
  3 P  1.01376180E+00  0.727159

  4 S  2.70005800E-01  1.000000

  5 P  2.70005800E-01  1.000000

  C (Carbon)
  ----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  3.04752490E+03  0.001835
  1 S  4.57369510E+02  0.014037
  1 S  1.03948690E+02  0.068843
  1 S  2.92101550E+01  0.232184
  1 S  9.28666300E+00  0.467941
  1 S  3.16392700E+00  0.362312

  2 S  7.86827240E+00 -0.119332
  2 S  1.88128850E+00 -0.160854
  2 S  5.44249300E-01  1.143456

  3 P  7.86827240E+00  0.068999
  3 P  1.88128850E+00  0.316424
  3 P  5.44249300E-01  0.744308

  4 S  1.68714400E-01  1.000000

  5 P  1.68714400E-01  1.000000

  H (Hydrogen)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  1.87311370E+01  0.033495
  1 S  2.82539370E+00  0.234727
  1 S  6.40121700E-01  0.813757

  2 S  1.61277800E-01  1.000000



 Summary of "ao basis" -> "ao basis" (spherical)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 O                           6-31G                   5        9   3s2p
 C                           6-31G                   5        9   3s2p
 H                           6-31G                   2        2   2s




 Summary of "ao basis" -> "ao basis" (spherical)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 O                           6-31G                   5        9   3s2p
 C                           6-31G                   5        9   3s2p
 H                           6-31G                   2        2   2s


  Caching 1-el integrals 
  tol_rho modified to match energy
  convergence criterion.

            General Information
            -------------------
          SCF calculation type: DFT
          Wavefunction type:  closed shell.
          No. of atoms     :     4
          No. of electrons :    16
           Alpha electrons :     8
            Beta electrons :     8
          Charge           :     0
          Spin multiplicity:     1
          Use of symmetry is: off; symmetry adaption is: off
          Maximum number of iterations: 500
          This is a Direct SCF calculation.
          AO basis - number of functions:    22
                     number of shells:    14
          Convergence on energy requested: 1.00D-06
          Convergence on density requested: 1.00D-05
          Convergence on gradient requested: 5.00D-04

              XC Information
              --------------
                        Slater Exchange Functional  1.000 local    
                      VWN V Correlation Functional  1.000 local    

             Grid Information
             ----------------
          Grid used for XC integration:  xfine     
          Radial quadrature: Euler-MacLaurin     
          Angular quadrature: Lebedev. 
          Tag              B.-S. Rad. Rad. Pts. Rad. Cut. Ang. Pts.
          ---              ---------- --------- --------- ---------
          O                   0.60       78           7.0      1454
          C                   0.70       75           6.0      1454
          H                   0.35       82           7.0      1202
          Grid pruning is: on 
          Number of quadrature shells:   317
          Spatial weights used:  Erf1

          Convergence Information
          -----------------------
          Convergence aids based upon iterative change in 
          total energy or number of iterations. 
          Levelshifting, if invoked, occurs when the 
          HOMO/LUMO gap drops below (HL_TOL): 1.00D-02
          DIIS, if invoked, will attempt to extrapolate 
          using up to (NFOCK): 10 stored Fock matrices.

                    Damping( 0%)  Levelshifting(0.5)       DIIS
                  --------------- ------------------- ---------------
          dE  on:    start            ASAP                start   
          dE off:    2 iters        500 iters           500 iters 


      Screening Tolerance Information
      -------------------------------
          Density screening/tol_rho: 1.00D-09
          AO Gaussian exp screening on grid/accAOfunc:  14
          CD Gaussian exp screening on grid/accCDfunc:  20
          XC Gaussian exp screening on grid/accXCfunc:  20
          Schwarz screening/accCoul: 1.00D-08


      Superposition of Atomic Density Guess
      -------------------------------------

 Sum of atomic energies:        -113.41107146

      Non-variational initial energy
      ------------------------------

 Total energy =    -114.013965
 1-e energy   =    -215.104686
 2-e energy   =      70.367627
 HOMO         =      -0.318547
 LUMO         =      -0.028582

   Time after variat. SCF:      0.1
   Time prior to 1st pass:      0.1

 Grid_pts file          = ./tddftgrad_ch2o.gridpts.0
 Record size in doubles =  12289        No. of grid_pts per rec  =   3070
 Max. records in memory =     43        Max. recs in file   =     90595

 Grid integrated density:      15.999999326816
 Requested integration accuracy:   0.10E-07

           Memory utilization after 1st SCF pass: 
           Heap Space remaining (MW):       12.58            12577140
          Stack Space remaining (MW):       13.11            13106916

   convergence    iter        energy       DeltaE   RMS-Dens  Diis-err    time
 ---------------- ----- ----------------- --------- --------- ---------  ------
 d= 0,ls=0.0,diis     1   -113.4212621425 -1.44D+02  1.23D-01  1.01D+00     0.4
 Grid integrated density:      15.999999561855
 Requested integration accuracy:   0.10E-07
 d= 0,ls=0.0,diis     2   -112.5044998618  9.17D-01  9.84D-02  9.06D+00     0.6
 Grid integrated density:      15.999999426402
 Requested integration accuracy:   0.10E-07
 d= 0,ls=0.0,diis     3   -113.5428950469 -1.04D+00  1.01D-02  2.58D-02     0.7
 Grid integrated density:      15.999999393964
 Requested integration accuracy:   0.10E-07
 d= 0,ls=0.0,diis     4   -113.5439887595 -1.09D-03  5.04D-03  1.57D-02     0.9
 d= 0,ls=0.0,diis     5   -113.5460126747 -2.02D-03  5.51D-04  1.13D-04     1.1
  Resetting Diis
 d= 0,ls=0.0,diis     6   -113.5460288846 -1.62D-05  3.89D-05  1.12D-06     1.3
 d= 0,ls=0.0,diis     7   -113.5460290467 -1.62D-07  1.34D-05  1.71D-08     1.5
 d= 0,ls=0.0,diis     8   -113.5460290272  1.95D-08  9.88D-06  2.05D-07     1.7


         Total DFT energy =     -113.546029027224
      One electron energy =     -216.148020874765
           Coulomb energy =       85.615101569617
    Exchange-Corr. energy =      -13.736204315183
 Nuclear repulsion energy =       30.723094593106

 Numeric. integr. density =       15.999999966503

     Total iterative time =      1.6s



                       DFT Final Molecular Orbital Analysis
                       ------------------------------------

 Vector    1  Occ=2.000000D+00  E=-1.864141D+01
              MO Center= -8.7D-03, -6.8D-01,  2.3D-13, r^2= 1.5D-02
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1      0.994225  1 O  s          

 Vector    2  Occ=2.000000D+00  E=-9.882645D+00
              MO Center=  4.0D-03,  5.4D-01, -3.2D-12, r^2= 2.8D-02
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    10      0.993829  2 C  s          

 Vector    3  Occ=2.000000D+00  E=-9.825689D-01
              MO Center= -4.7D-03, -3.0D-01,  2.0D-10, r^2= 4.8D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      0.423861  1 O  s                  6      0.393041  1 O  s          
    11      0.228817  2 C  s                  4      0.199650  1 O  py         
    13     -0.199746  2 C  py                 1     -0.193907  1 O  s          

 Vector    4  Occ=2.000000D+00  E=-5.728467D-01
              MO Center=  6.2D-03,  7.5D-01,  1.4D-11, r^2= 1.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    15      0.351340  2 C  s                 11      0.339752  2 C  s          
     6     -0.254521  1 O  s                 13      0.224448  2 C  py         
     2     -0.196323  1 O  s                 19      0.176600  3 H  s          
    21      0.176672  4 H  s                 10     -0.165653  2 C  s          

 Vector    5  Occ=2.000000D+00  E=-4.461883D-01
              MO Center=  3.3D-03,  4.8D-01,  1.0D-13, r^2= 1.3D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    12      0.460938  2 C  px                 3      0.266863  1 O  px         
    16      0.205718  2 C  px                19     -0.186735  3 H  s          
    21      0.186710  4 H  s          

 Vector    6  Occ=2.000000D+00  E=-3.888765D-01
              MO Center= -7.7D-03, -5.8D-01, -2.9D-09, r^2= 1.0D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      0.514319  1 O  py                 6     -0.399212  1 O  s          
    13     -0.368351  2 C  py                 8      0.247864  1 O  py         
     2     -0.196519  1 O  s          

 Vector    7  Occ=2.000000D+00  E=-3.693104D-01
              MO Center= -3.9D-03, -2.2D-01,  2.5D-09, r^2= 1.0D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5      0.510030  1 O  pz                14      0.378213  2 C  pz         
     9      0.305631  1 O  pz                18      0.221420  2 C  pz         

 Vector    8  Occ=2.000000D+00  E=-2.198164D-01
              MO Center= -3.8D-03, -2.0D-01, -8.5D-14, r^2= 1.6D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      0.595511  1 O  px                 7      0.422293  1 O  px         
    20      0.314574  3 H  s                 22     -0.314580  4 H  s          
    19      0.183548  3 H  s                 21     -0.183515  4 H  s          

 Vector    9  Occ=0.000000D+00  E=-1.005519D-01
              MO Center= -3.1D-04,  1.3D-01,  3.3D-09, r^2= 1.3D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    18      0.529161  2 C  pz                14      0.523507  2 C  pz         
     5     -0.468720  1 O  pz                 9     -0.443464  1 O  pz         

 Vector   10  Occ=0.000000D+00  E= 6.827306D-02
              MO Center=  1.1D-02,  1.2D+00, -7.8D-09, r^2= 2.8D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    15      1.802709  2 C  s                 20     -1.304625  3 H  s          
    22     -1.304710  4 H  s                 17      0.554044  2 C  py         
    13      0.242859  2 C  py                11      0.236898  2 C  s          

 Vector   11  Occ=0.000000D+00  E= 1.397550D-01
              MO Center=  9.9D-03,  1.1D+00, -1.2D-13, r^2= 2.8D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    20      1.477381  3 H  s                 22     -1.476949  4 H  s          
    16      1.242143  2 C  px                12      0.570026  2 C  px         
     7     -0.352957  1 O  px                 3     -0.195693  1 O  px         

 Vector   12  Occ=0.000000D+00  E= 1.730445D-01
              MO Center=  2.1D-03,  3.6D-01, -2.4D-09, r^2= 2.4D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    17      1.868005  2 C  py                 6      1.829136  1 O  s          
    15     -1.369814  2 C  s                  8      0.822336  1 O  py         
    22     -0.255289  4 H  s                  4      0.253970  1 O  py         
    20     -0.250922  3 H  s                 13      0.170953  2 C  py         
     2      0.164599  1 O  s          

 Vector   13  Occ=0.000000D+00  E= 4.556481D-01
              MO Center=  4.8D-03,  6.2D-01,  2.2D-08, r^2= 1.9D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    18      1.184945  2 C  pz                14     -1.031960  2 C  pz         

 Vector   14  Occ=0.000000D+00  E= 5.733904D-01
              MO Center=  5.7D-05,  2.0D-01,  4.1D-13, r^2= 2.1D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    16      2.247513  2 C  px                20      0.959289  3 H  s          
    22     -0.960932  4 H  s                 12     -0.791339  2 C  px         
     3     -0.384514  1 O  px                21     -0.284800  4 H  s          
    19      0.283049  3 H  s          

 Vector   15  Occ=0.000000D+00  E= 5.768325D-01
              MO Center=  4.3D-03,  5.2D-01, -3.1D-09, r^2= 1.5D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    17     -1.188594  2 C  py                13      1.088303  2 C  py         
    15     -0.873192  2 C  s                  4      0.397479  1 O  py         
     8      0.374638  1 O  py                20      0.298802  3 H  s          
    19      0.295427  3 H  s                 21      0.294283  4 H  s          
    22      0.293456  4 H  s                  2      0.230454  1 O  s          

 Vector   16  Occ=0.000000D+00  E= 6.636526D-01
              MO Center=  5.4D-03,  7.1D-01, -3.4D-09, r^2= 1.7D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    15      1.895798  2 C  s                 11     -1.328472  2 C  s          
    19     -0.628534  3 H  s                 21     -0.628353  4 H  s          
    13      0.382195  2 C  py                17      0.284687  2 C  py         
     4      0.237339  1 O  py                20     -0.161707  3 H  s          
    22     -0.161385  4 H  s          

 Vector   17  Occ=0.000000D+00  E= 8.058889D-01
              MO Center=  8.0D-03,  1.0D+00,  1.3D-13, r^2= 2.3D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    20      1.461850  3 H  s                 22     -1.460902  4 H  s          
    16      1.336690  2 C  px                19     -0.789847  3 H  s          
    21      0.789254  4 H  s                 12     -0.731962  2 C  px         
     7     -0.495134  1 O  px                 3      0.349036  1 O  px         

 Vector   18  Occ=0.000000D+00  E= 8.240094D-01
              MO Center=  3.6D-03,  4.7D-01, -1.2D-08, r^2= 2.9D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    15      2.768239  2 C  s                 20     -1.216663  3 H  s          
    22     -1.217628  4 H  s                 11     -1.079126  2 C  s          
     8     -0.870710  1 O  py                19      0.537145  3 H  s          
    21      0.537633  4 H  s                  4      0.511416  1 O  py         
    13     -0.437162  2 C  py                17      0.437564  2 C  py         


 center of mass
 --------------
 x =  -0.00447957 y =  -0.13292483 z =   0.00000000

 moments of inertia (a.u.)
 ------------------
          48.413971884362          -0.434990612490           0.000000000000
          -0.434990612490           6.423618600421           0.000000000000
           0.000000000000           0.000000000000          54.837590484783

     Multipole analysis of the density
     ---------------------------------

     L   x y z        total         alpha         beta         nuclear
     -   - - -        -----         -----         ----         -------
     0   0 0 0      0.000000     -8.000000     -8.000000     16.000000

     1   1 0 0      0.009446      0.028945      0.028945     -0.048445
     1   0 1 0      0.909327      0.381441      0.381441      0.146445
     1   0 0 1      0.000000      0.000000      0.000000      0.000000

     2   2 0 0     -8.658938     -7.515400     -7.515400      6.371862
     2   1 1 0     -0.008035     -0.121619     -0.121619      0.235203
     2   1 0 1      0.000000      0.000000      0.000000      0.000000
     2   0 2 0     -9.194986    -19.153015    -19.153015     29.111045
     2   0 1 1      0.000000      0.000000      0.000000      0.000000
     2   0 0 2     -8.641746     -4.320873     -4.320873      0.000000

                                NWChem TDDFT Module
                                -------------------


            General Information
            -------------------
           No. of orbitals :    44
            Alpha orbitals :    22
             Beta orbitals :    22
        Alpha frozen cores :     0
         Beta frozen cores :     0
     Alpha frozen virtuals :     0
      Beta frozen virtuals :     0
         Spin multiplicity :     1
    Number of AO functions :    22
        Use of symmetry is : off
      Symmetry adaption is : off
         Schwarz screening : 0.10D-07

              XC Information
              --------------
                 Slater Exchange Functional   1.00 local    
               VWN V Correlation Functional   1.00 local    

             TDDFT Information
             -----------------
          Calculation type : TDDFT             
         Wavefunction type : Restricted triplets
          No. of electrons :    16
           Alpha electrons :     8
            Beta electrons :     8
              No. of roots :     2
          Max subspacesize :  4400
            Max iterations :   100
               Target root :     1
           Target symmetry : none
      Symmetry restriction : off
                 Algorithm : Incore multiple tensor contraction
        Davidson threshold : 0.10D-03

            Memory Information
            ------------------
          Available GA space size is         104857116 doubles
          Available MA space size is          26212868 doubles
          Length of a trial vector is          112
          Estimated peak GA usage is           1981328 doubles
          Estimated peak MA usage is              3000 doubles

    2 smallest eigenvalue differences (eV) 
--------------------------------------------------------
  No. Spin  Occ  Vir  Irrep   E(Occ)    E(Vir)   E(Diff)
--------------------------------------------------------
    1    1    8    9 a        -0.220    -0.101     3.245
    2    1    7    9 a        -0.369    -0.101     7.313
--------------------------------------------------------

  Entering Davidson iterations
  Restricted triplet excited states

  Iter   NTrls   NConv    DeltaV     DeltaE      Time   
  ----  ------  ------  ---------  ---------  --------- 
    1      2       0     0.72E-01   0.10+100        0.5
    2      4       0     0.12E-01   0.29E-02        0.5
    3      8       1     0.48E-02   0.11E-03        0.7
    4     10       1     0.10E-02   0.14E-04        0.5
    5     12       1     0.33E-03   0.36E-06        0.5
    6     14       1     0.16E-03   0.23E-07        0.4
    7     16       2     0.81E-04   0.69E-08        0.5
  ----  ------  ------  ---------  ---------  --------- 
  Convergence criterion met

  Ground state a        -113.546029027224 a.u.

  ----------------------------------------------------------------------------
  Root   1 triplet a              0.107588003 a.u.                2.9276 eV 
  ----------------------------------------------------------------------------
     Transition Moments                    Spin forbidden
     Oscillator Strength                   Spin forbidden

     Occ.    8  a   ---  Virt.    9  a   -1.00128 X
     Occ.    8  a   ---  Virt.    9  a   -0.05154 Y
  ----------------------------------------------------------------------------
  Root   2 triplet a              0.220126199 a.u.                5.9899 eV 
  ----------------------------------------------------------------------------
     Transition Moments                    Spin forbidden
     Oscillator Strength                   Spin forbidden

     Occ.    7  a   ---  Virt.    9  a   -1.00371 X
     Occ.    7  a   ---  Virt.    9  a   -0.09985 Y

              Target root =      1
          Target symmetry = none
      Ground state energy =   -113.546029027224
        Excitation energy =      0.107588003388
     Excited state energy =   -113.438441023836

 fn_civecs: ./tddftgrad_ch2o.civecs_triplet                                                                                                                                                                                                                                

  CI vectors are stored in ./tddftgrad_ch2o.civecs_triplet 


                           NWChem TDDFT Gradient Module
                           ----------------------------


 ./tddftgrad_ch2o.civecs_triplet                                                                                                                                                                                                                                
 Calculated gradients of:
 Number of Triplet roots   1
 Triplet roots    1

 Start at time  cpu:        5.2s  wall:        7.1s

TDDFT Energy Check(  1) =       0.10758800353234


Iterative solution of linear equations
  No. of variables      112
  No. of equations        1
  Maximum subspace       40
        Iterations      250
       Convergence  1.0D-04
        Start time      9.9


   iter   nsub   residual    time
   ----  ------  --------  ---------
     1      1    1.21D+00      10.4
     2      2    3.48D-02      10.9
     3      3    1.38D-02      11.4
     4      4    1.14D-03      11.8
     5      5    5.41D-05      12.3
     Dipole Moment    X  0.05596   Y  0.58350   Z  0.00000

     No. of electrons (tr(P*S)):     0.1600000E+02



                         Root   1

                         TDDFT ENERGY GRADIENTS

    atom               coordinates                        gradient
                 x          y          z           x          y          z
   1 O      -0.016488  -1.294429   0.000000    0.001210   0.118367   0.000000
   2 C       0.007588   1.029905   0.000000   -0.001359  -0.135175   0.000000
   3 H      -1.765501   2.179267   0.000000   -0.006839   0.008469   0.000000
   4 H       1.803427   2.143183   0.000000    0.006989   0.008339   0.000000

 TDDFT Gradient time  cpu:        8.2s  wall:       10.5s

 Task  times  cpu:       13.4s     wall:       17.4s


                                NWChem Input Module
                                -------------------


 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:  822      825     1.42e+05 4298     3.00e+04  104        0     1327     
number of processes/call 1.03e+00 1.32e+00 1.34e+00 2.18e+00 0.00e+00
bytes total:             1.59e+07 1.08e+06 6.17e+06 8.72e+04 0.00e+00 1.06e+04
bytes remote:            2.96e+06 1.97e+05 3.64e+06 -4.78e+04 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 3018408 bytes

MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        22	        97
	current total bytes		         0	         0
	maximum total bytes		   4240480	  22510952
	maximum total K-bytes		      4241	     22511
	maximum total M-bytes		         5	        23


                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:

                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018

                              AUTHORS & CONTRIBUTORS
                              ----------------------
          E. Apra, E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski,
       T. P. Straatsma, M. Valiev, H. J. J. van Dam, D. Wang, T. L. Windus,
    J. Hammond, J. Autschbach, K. Bhaskaran-Nair, J. Brabec, K. Lopata, W. Ma,
       M. Klemm, O. Villa, F. Aquino, S. Hirata, M. T. Hackler, T. Risthaus,
       M. Malagoli, A. Marenich, A. Otero-de-la-Roza, J. Mullin, P. Nichols,
      R. Peverati, J. Pittner, Y. Zhao, P.-D. Fan, A. Fonari, R. J. Harrison,
      M. Dupuis, D. Silverstein, D. M. A. Smith, J. Nieplocha, V. Tipparaju,
    M. Krishnan, B. E. Van Kuiken, A. Vazquez-Mayagoitia, L. Jensen, M. Swart,
      Q. Wu, T. Van Voorhis, A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown,
      G. Cisneros, G. I. Fann, H. Fruchtl, J. Garza, K. Hirao, R. A. Kendall,
      J. A. Nichols, K. Tsemekhman, K. Wolinski, J. Anchell, D. E. Bernholdt,
      P. Borowski, T. Clark, D. Clerc, H. Dachsel, M. J. O. Deegan, K. Dyall,
    D. Elwood, E. Glendening, M. Gutowski, A. C. Hess, J. Jaffe, B. G. Johnson,
     J. Ju, R. Kobayashi, R. Kutteh, Z. Lin, R. Littlefield, X. Long, B. Meng,
      T. Nakajima, S. Niu, L. Pollack, M. Rosing, K. Glaesemann, G. Sandrone,
      M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe, A. T. Wong, Z. Zhang.

 Total times  cpu:       13.4s     wall:       17.6s
